### PR TITLE
New overview graphic in light of asset refactor.

### PR DESCRIPTION
**Pull request by danielballan**
Now that the asset refactor is merged, I asked Tiffany to update our
overview graphic: removing the arrow representing ophyd inserting into the
asset registry.

### CODE REVIEW

File: source/sphinxext/xfig.py [hunk 1]
```diff
@@ -1,172 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (tacaswell) [inline source/sphinxext/xfig.py]: I protest to removing our xfig extension.

Comment (danielballan) [reply source/sphinxext/xfig.py]: This vendored version is no longer compatible with sphinx. If you want to it have handy here for future possible use, that's fine, but would you take on the task of vendoring a working version?

Comment (tacaswell) [reply source/sphinxext/xfig.py]: This is the upstream of this extension.

Comment (tacaswell) [reply source/sphinxext/xfig.py]: If I fix it can we use xfig to make diagrams again? ;)

Comment (danielballan) [reply source/sphinxext/xfig.py]: Nope, only Dropbox Paper :- D

**Comment by danielballan** [comment]
You might notice that I nuked `tutorials.rst`. It was mostly advertising (dates etc.) for in-person tutorials given two years ago. A lot of the intersphinx rst links were busted, and I decided it wasn't worth fixing. Also, the existence of this file has sometimes confused people who are looking for https://nsls-ii.github.io/bluesky/tutorial.html.

The only useful thing in there was a link to the IPython cheatsheet, which we can revisit along with the other ongoing cheatsheet work.

@tacaswell Wanna fix your extension and push it here?

**Pull request merged**